integration_tests/pages/requestMethod.ts

```diff
@@ -0,0 +1,13 @@
+import Page, { PageElement } from './page'
+
+export default class RequestMethodPage extends Page {
+  constructor() {
+    super('What method was used to make this request?')
+  }
+
+  checkbox1 = (): PageElement => cy.get('#method')
+
+  textValue1 = (): PageElement => cy.get(':nth-child(1) > .govuk-label')
```

Comment:
Suggestion (untested!) - and you'd need to update the test script...

```suggestion
  textValue = (index: number): PageElement => cy.get(':nth-child(index) > .govuk-label')
```

Reply:
See previous comment, would only work for `textValue` not `checkbox`. 
Also currently only testing the page works, so not using anything other than the first textbox / checkbox